sys/net/application_layer/nanocoap/nanocoap.c

```diff
@@ -280,7 +280,7 @@ uint8_t *coap_iterate_option(coap_pkt_t *pkt, uint8_t **optpos,
     }
 }
 
-unsigned coap_get_content_type(coap_pkt_t *pkt)
+static unsigned _get_content_format(coap_pkt_t *pkt, unsigned int opt_num)
 {
     uint8_t *opt_pos = coap_find_option(pkt, COAP_OPT_CONTENT_FORMAT);
```

Comment:
This should be
```suggestion
    uint8_t *opt_pos = coap_find_option(pkt, opt);
```

no?

Reply:
You are right, thanks :)

Reply:
Should have been `opt_num` :sweat_smile:

Reply:
Oh, sorry, I just hit the commit button :sweat_smile: Should now be the correct variable name :)

Reply:
Also sorry for the mistake in the first place ^^"